---
layout: index_page
category: science
title: Science
---

<span id="virus" />
### Virus

Respiratory viruses impose a significant public health burden, in large part due to their rapid evolution and ability to evade host immunity. Influenza, COVID-19, and now RSV infection, are all vaccine-preventable diseases, yet our current system of genomic surveillance is limited by significant knowledge gaps in forecasting new antigenic drift mutations and remarkably little attention to determinants of within-host and population-level viral fitness outside of antigenic sites. The objective of this project is to combine large-scale genomic surveillance and functional genomics to define determinants of epidemic success in three respiratory viruses. This project will take an integrated bidirectional approach, in which genomic surveillance is used to identify strains and mutations for experimental analysis, and in which functional genomics data are used to improve population-level inference of adaptive viral evolution. We will develop a Bayesian model to identify SARS-CoV-2 mutations positively selected within hosts and related these to ongoing surveillance data. For RSV, we will use mutational antigenic profiling of the fusion (F) protein to define and anticipate antigenic drift. We will take a population-scale approach to influenza virus by applying phylodynamic models to regional whole genome surveillance data to identify strains and mutations conferring a population-level fitness advantage. While different approaches are taken for each virus, they are universally applicable and we expect that together they will provide a foundation for more accurate prediction of emerging strains and the development of more protective vaccines.

{% include topic_bibliography.html topic="virus"  n=5 %}

[See more...](virus)


<span id="bacteria" />
### Bacteria
Carbapenem-resistant Enterobacterales (CRE) has been designated as an urgent antibiotic resistance threat by both the U.S. Centers for Disease Control and Prevention and the World Health Organization. The basis for this notoriety is the propensity for CRE to cause outbreaks among vulnerable hospitalized patients, high mortality rates for infected patients, and the emergence of strains of CRE that are resistant to virtually all available antibiotics. The most problematic strains of CRE are those that have become carbapenem-resistant via the acquisition of a carbapenemase gene that can cleave most beta-lactam antibiotics. These carbapenemase genes are most often carried on plasmids, and through conjugation have disseminated widely among diverse members of the Enterobacterales order. Given the great diversity of CRE observed in healthcare settings, a critical need from a public health surveillance perspective is the ability to assess the risk posed by a newly detected strain. We reason that the risk of a newly emerged CRE strain is dictated by: i) the stability of its association with the carbapenemase containing plasmid, ii) itâ€™s capacity to disseminate this plasmid to other Enterobacterales, iii) the potential of the CRE strain to evolve resistance to additional front-line antibiotics, and iv) its potential to colonize, infect and spread among vulnerable patient populations. In this project we experimentally quantify the variation in these four determinants of risk in longitudinal collections of CRE collected from two healthcare networks in the U.S. and employ bacterial genome-wide association studies (bGWAS) to identify genetic variation associated with each of these four phenotypes. We plan to take advantage of the longitudinal nature our sample collections, and the availability of relevant patient meta-data, to quantify the degree to which variation in risk potential predicts patterns of CRE strain emergence, resistance evolution and spread. We expect this project to contribute substantially to our understanding of how genetic variation in CRE influences its epidemic potential.

{% include topic_bibliography.html topic="bacteria"  n=5 %}

[See more...](bacteria)


<span id="fungus" />
### Fungus

A high skin fungal burden is a defining feature of *Candida auris*, and this burden is associated with increased risk of disease. The first step in this process is the ability of strains to attach and adhere, on both biotic and abiotic surfaces. This attachment also is the first step in the formation of biofilms, which are linked with both persistence and antifungal drug resistance. However, the cell biology and genetic circuitry underlying the ability of *C. auris* to colonize the skin are not known. Our goal is to identify the genetic determinants and regulation of *C. auris* skin colonization and thus persistence and transmission between individuals. We have developed genetic tools for interrogating this organism, giving us unprecedented power to analyze the genes required for *C. auris* colonization. We will use a population genetics and genome-wide association approach to identify naturally-occurring variants that are associated with increased attachment. We complement these studies by performing forward genetics screens to uncover new effectors and regulators of *C. auris* skin attachment and by experimental evolution to identify variants of concern for skin colonization. Together, these studies will advance new concepts for how *C. auris* can regulate adherence, colonization, and outbreak potential. 

{% include topic_bibliography.html topic="fungus"  n=5 %}

[See more...](fungus)


<span id="bioinformatics" />
### Technology and Data Core

Each project in the MIDGE Center generates large genomic compendia, which are analyzed in the context of rich phenotypic measurements. The Technology and Data Core supports these projects and the overall mission of MIDGE by providing intellectual capacity, technical skill, and infrastructure to enable genomic and phenotypic data generation, analysis and dissemination of data, resources, and results to the research community. Specific activities of the core include: facilitating genomic data generation, quality control and rapid dissemination of cleaned sequence data, providing capacity for extraction of genetic variation from cleaned sequencing data, providing support for laboratory automation, high-throughput phenotyping and integration of phenotypic and genotypic data through advanced phylogenetic analysis and mGWAS, and ensuring consistent and uniform sharing of data, protocols and software emanating from MIDGE.

{% include topic_bibliography.html topic="technology"  n=5 %}

[See more...](technology)


